main.go

```diff
@@ -686,6 +728,81 @@ func selectScheduler(ctx context.Context, monitor metrics.Service) (scheduler.Se
 	return scheduler, nil
 }
 
+// startFeeRecipientProvider starts the appropriate fee recipient provider given user input.
+func startFeeRecipientProvider(ctx context.Context, monitor metrics.Service, majordomo majordomo.Service) (feerecipientprovider.Service, error) {
+	addr := viper.GetString("feerecipient.default-address")
+	if addr == "" {
```

Comment:
Worth having a check for the 0x000000000...001 address in case the configuration has been put together lazily?